main.py

```diff
@@ -66,6 +66,9 @@ def set_postcode():
 
     return flask.redirect("/candidates")
 
+#####################################################################
+# Clear data
+
 @app.route('/clear_postcode')
```

Comment:
Nothing else seems to use clear_postcode now, so you could remove it completely, merge it into clear_all